### DISCUSSION TITLE
What happens if I power off during creating VM installation and powering it back on

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#8888] [Unsupported Versions] [answered]
Hi, I am wondering if powering back on later picks up where it left off? I hope so. When I powered up that VM, I just see a blinking cursor? But is it at the point where it left off?
 I read that the total installation time is about 2 hrs. In fact I have experianced 1 1/2 hours.  
 

Any links that could help me? 

   thanks for any help or suggestions

**Comment by dougburks** [comment, marked as answer]
If you power off a VM, it's the same as pulling the power from a physical machine. If you suspend a VM, you can resume from that suspend point. You can read more by consulting the documentation for whatever virtualization software you're using.

**Reply by iqworks** [reply]
thanks Doug
